frame/nomination-pools/src/lib.rs

```diff
@@ -1807,10 +1868,28 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			let bonded_pool = BondedPool::<T>::get(pool_id).ok_or(Error::<T>::PoolNotFound)?;
 			ensure!(bonded_pool.can_nominate(&who), Error::<T>::NotNominator);
-			T::StakingInterface::nominate(bonded_pool.bonded_account(), validators)?;
-			Ok(())
+			T::StakingInterface::nominate(bonded_pool.bonded_account(), validators)
 		}
 
+		/// Chill on behalf of the pool.
+		///
+		/// The dispatch origin of this call must be signed by the pool nominator or the pool
+		/// root role, same as [`Pallet::nominate`].
+		///
+		/// This directly forward the call to the staking pallet, on behalf of the pool bonded
+		/// account.
+		#[pallet::weight(T::WeightInfo::chill())]
+		pub fn chill(origin: OriginFor<T>, pool_id: PoolId) -> DispatchResult {
```

Comment:
transaction ordering? Should move this to the bottom or use the new feature keith made